tests/extension/oneapi-current-device/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(SYCL_CTS_ENABLE_EXT_ONEAPI_ROOT_GROUP_TESTS)
```

Comment:
Not `SYCL_CTS_ENABLE_EXT_ONEAPI_ROOT_GROUP_TESTS`, should be `SYCL_CTS_ENABLE_EXT_ONEAPI_CURRENT_DEVICE_TESTS`.
You also need to register `SYCL_CTS_ENABLE_EXT_ONEAPI_CURRENT_DEVICE_TESTS` in the main `CMakeLists.txt`